<ui version="4.0" >
 <class>SearchWindowClass</class>
 <widget class="QWidget" name="SearchWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Contact</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Protocol:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="protoCmb" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Account:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="accountCmb" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page1" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>473</width>
        <height>256</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Please select a protocol and an account above.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
